app/controllers/miq_policy_set_controller.rb

```diff
@@ -83,7 +83,7 @@ def profile_cancel
   end
 
   def profile_save_add
-    assert_privileges("profile_#{params[:id] ? "edit" : "new"}")
+    assert_privileges("miq_policy_set#{params[:id] ? "edit" : "new"}")
```

Comment:
This prevented policy profiles from being saved for non-admin users.

Reply:
This looks wrong - Is it `miq_policy_setedit` or `miq_policy_set_edit` ?

Reply:
Good eye.  Let me check the UI.  I think you're right.